modules/sumologic-configuring-acs.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-sumologic.adoc
+:_module-type: PROCEDURE
+[id="sumologic-configuring-acs_{context}"]
+= Configuring {product-title}
+
+Create a new integration in {product-title} by using the HTTP URL.
+
+.Procedure
+. On the RHACS portal, navigate to *Platform Configuration* -> *Integrations*.
+. Scroll down to the *Notifier Integrations* section and select *Sumo Logic*.
+. Click *New Integration* (add icon).
```

Comment:
I think you want to markup as `(`**add**` icon)` and similar for the rest of the PR.